### PR TITLE
Added files to support URIv18 mesh

### LINKED ISSUE #860: Add support for URI mesh(es)

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #860

### CODE REVIEW

File: config/mesh_defaults.sh [hunk 1]
```diff
@@ -369,6 +369,58 @@ case $GRIDNAME in
       # unit offset url https://asgs-static-assets.sfo2.digitaloceanspaces.com/offsets/unit_offset_nc_inundation_v9.99_rivers.dat.xz
       UNITOFFSETFILE=unit_offset_nc_inundation_v9.99_rivers.dat
       ;;
+"uriv18"|"URIv18")
```
Comment (wwlwpd) [inline config/mesh_defaults.sh:372]: I think we're missing here:
```
nodes=...
elements=...
```

File: config/mesh_defaults.sh [hunk 2]
```diff
@@ -369,6 +369,58 @@ case $GRIDNAME in
...
+      # intersection between mesh, models, hpc platform, and number of compute cores:
+      HINDCASTWALLTIME="24:00:00" # hindcast wall clock time
+      ADCPREPWALLTIME="02:00:00"  # adcprep wall clock time, including partmesh
+      NOWCASTWALLTIME="07:00:00"  # longest nowcast wall clock time
+      FORECASTWALLTIME="07:00:00" # forecast wall clock time
+      # unit offset url https://asgs-static-assets.sfo2.digitaloceanspaces.com/offsets/unit_offset_hsofs.dat.xz
+      UNITOFFSETFILE=unit_offset_hsofs.dat
+      ;;
+   "hsofs_NE-hires_v2_depf2")
+      INPUTDIR=$SCRIPTDIR/input/meshes/hsofs_NE-hires_v2_depf2
```
Comment (wwlwpd) [inline config/mesh_defaults.sh:399]: missing, `elements` and `nodes`

**Review by wwlwpd — CHANGES_REQUESTED** [review]
missing element and node fields for both meshes

File: config/mesh_defaults.sh (hunk 1 above)
Comment (jasonfleming) [reply config/mesh_defaults.sh:372]: Good catch.

**Comment by wwlwpd** [comment]
When you address my comments in the commits and are ready for another review, please mark these mesh commits as "ready for review" so I know they are read. Thanks!

**Comment by jasonfleming** [comment]
Will do. Haven't even run this mesh even once yet, so not sure how long it will be ...

File: config/mesh_defaults.sh (hunk 1 above)
Comment (jasonfleming) [reply config/mesh_defaults.sh:372]: fixed this

File: config/mesh_defaults.sh (hunk 2 above)
Comment (jasonfleming) [reply config/mesh_defaults.sh:403]: fixed this

**Review by wwlwpd — APPROVED** [review]
Will be merging manually after squash.

**Pull request closed without merging**